docs/AboutUs.md

```diff
@@ -28,7 +28,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](team/lywich.md)]
 
 * Role: Developer
-* Responsibilities: tbc
+* Responsibilities: Refactoring + Scheduling + Integration
```

Comment:
Perhaps you could add more details like 'Refactoring AB3 to fit Clock-Work's implementation', and be more specific about what you mean by scheduling and integration? You could also add information about the features you were in-charge of implementing (e.g. sort)